Mergin/projects_manager.py

```diff
@@ -49,7 +49,12 @@ def have_writing_permissions(self, project_name):
         info = self.mc.project_info(project_name)
         username = self.mc.username()
         writersnames = info["access"]["writersnames"]
-        return username in writersnames
+        permissions = info["permissions"]
```

Comment:
we do pretty much the same in py-client code, wondering if it is not better to create function only there